basic-integration/surfly-integration-with-tag-manager.md

```diff
@@ -0,0 +1,177 @@
+      start_docked: true,
+      cookie_transfer_enabled: true,
+      cookie_transfer_proxying: false,
+      splash: false
+    };
+//Code that shows the session-id so that it can be passed to the helpdesk agent
+    Surfly.session(settings)
+    .on('session_started', function(session, event) {
+      document.getElementById("id-cover").style.display = "none";
+      console.log(session);
+      var showId = document.getElementById("btn-start-session");
+      showId.style.display = "block";
+      showId.textContent = session.pin;
+    }).startLeader();
+  }
+
+//Add your chat box snippet code (for this example I have used Olark)
+  if(!window.__surfly){
+  
+    //REPLACE THIS PART WITH YOUR OWN CHAT BOX SNIPPET
+      window.olark||(function(c){var f=window,d=document,l=f.location.protocol=="https:"?"https:":"http:",z=c.name,r="load";var nt=function(){
+      f[z]=function(){
+      (a.s=a.s||[]).push(arguments)};var a=f[z]._={
+      },q=c.methods.length;while(q--){(function(n){f[z][n]=function(){
+      f[z]("call",n,arguments)}})(c.methods[q])}a.l=c.loader;a.i=nt;a.p={
+      0:+new Date};a.P=function(u){
+      a.p[u]=new Date-a.p[0]};function s(){
+      a.P(r);f[z](r)}f.addEventListener?f.addEventListener(r,s,false):f.attachEvent("on"+r,s);var ld=function(){function p(hd){
+      hd="head";return["<",hd,"></",hd,"><",i,' onl' + 'oad="var d=',g,";d.getElementsByTagName('head')[0].",j,"(d.",h,"('script')).",k,"='",l,"//",a.l,"'",'"',"></",i,">"].join("")}var i="body",m=d[i];if(!m){
+      return setTimeout(ld,100)}a.P(1);var j="appendChild",h="createElement",k="src",n=d[h]("div"),v=n[j](d[h](z)),b=d[h]("iframe"),g="document",e="domain",o;n.style.display="none";m.insertBefore(n,m.firstChild).id=z;b.frameBorder="0";b.id=z+"-loader";if(/MSIE[ ]+6/.test(navigator.userAgent)){
+      b.src="javascript:false"}b.allowTransparency="true";v[j](b);try{
+      b.contentWindow[g].open()}catch(w){
+      c[e]=d[e];o="javascript:var d="+g+".open();d.domain='"+d.domain+"';";b[k]=o+"void(0);"}try{
+      var t=b.contentWindow[g];t.write(p());t.close()}catch(x){
+      b[k]=o+'d.write("'+p().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};ld()};nt()})({
+      loader: "static.olark.com/jsclient/loader0.js",name:"olark",methods:["configure","extend","declare","identify"]});
+      /* custom configuration goes here (www.olark.com/documentation) */
+      olark.identify('** add your own Olark identifier **');  
+    //UNTIL HERE
+    
+  }
+
+//Function that ends the session, so that it can be applied to the "end Session"-button
+  function sessionEnd() {
+    Surfly.currentSession.end();
+  }
+</script>
+```
+{% em color="#ffffe0" %} Google Tag Manager puts everything in an iframe, and adds `<style =“display: none; visibility: hidden;”>` to the code you inject. Make sure you remove this attribute within the tag (as shown in the example above), otherwise the divs you've added will not appear. {% endem %}
```

Comment:
I noticed that quotes in the tag are incorrect (they are typographical curly double quotes instead of normal double quotes). Probably broke during copypasting